source/Octopus.Data/Resources/SensitiveValue.cs

```diff
@@ -7,6 +7,7 @@ public class SensitiveValue : IEquatable<SensitiveValue>
     {
         public bool HasValue { get; set; }
         public string? NewValue { get; set; }
+        public string? Hint { get; set; }
```

Comment:
Wondering if we should compare this property in its `Equals` method or not

Reply:
I actually don't know where these would be used, but from a look at the compare methods I think they may be wrong. Or at least very misleading. If I'm reading right, if 2 things that had different values had been loaded they'd both have `HasValue == true` and they would also both have null NewValue. So they would look the same, even though they are not. I think we should actually remove all of this.